infrastructure/quick-deploy/aws/storage/parameters.tfvars

```diff
@@ -124,7 +124,8 @@ mongodb = {
   tag                = "6.0.1"
   node_selector      = { "grid/type" = "Operator" }
   image_pull_secrets = ""
-  persistent_volume  = null
+  #persistent_volume  = null
```

Comment:
uncomment this line. By default, EFS is optional as persistent volume for MongoDB.